components/brave_component_updater/browser/brave_component.h

```diff
@@ -19,6 +19,8 @@ class PrefService;
 
 namespace brave_component_updater {
 
+std::string GetUpdateURLHost();
```

Comment:
this is not a good place to put this, we should be able to have a dep for `GetUpdateURLHost` without including all of `components/brave_component_updater/browser`